inaugurator/mount.py

```diff
@@ -32,6 +34,15 @@ def mountRoot(self):
         sh.run("/usr/sbin/busybox umount %s" % self._ROOT_MOUNT_POINT)
 
     @contextlib.contextmanager
+    def mountOsmosisCache(self):
```

Comment:
copy paste from mountRoot. Don't u want to implement one function and call it both from mountRoot and and mountOsmosisCache?